### PR TITLE
remove empty dictionary build calls

### PR DESCRIPTION
**Pull request by rlcee**
code cleanup only, no change to build output

### CODE REVIEW

File: scripts/build/python/mu2e_helper.py [hunk 1]
```diff
@@ -140,29 +140,28 @@ def make_plugins( self, userlibs, exclude_cc = [], cppf = [], pf = [] ):
     #   Make the dictionary and rootmap plugins.
     #
     def make_dict_and_map( self, userlibs=[], pf_dict=[] ):
-        if os.path.exists('classes.h') and os.path.exists('classes_def.xml'):
```
Comment (kutschke) [inline scripts/build/python/mu2e_helper.py:143]: What happens if someone (inappropriately) adds a make_dict_and_map stanza to an SConscript file in a directory that does not have classes.h and classes_def.xml files?   Will there be an understandable error message?  Silently do nothing?

**Comment by kutschke** [comment]
Ray and I discussed this PR before he did the work.  I have assigned it to myself and I don't think we need additional reviewers.

File: scripts/build/python/mu2e_helper.py (hunk 1 above)
Comment (rlcee) [reply scripts/build/python/mu2e_helper.py:143]: you get a message like "don't know how to build classes.h required for .."

Comment (kutschke) [reply scripts/build/python/mu2e_helper.py:143]: Thanks. Does it stop the build or is it informational only?

Comment (rlcee) [reply scripts/build/python/mu2e_helper.py:143]: It is a fatal scons error and the build stops

Comment (kutschke) [reply scripts/build/python/mu2e_helper.py:143]: Good.  That's what I hoped.  I will merge.

**Review by kutschke — APPROVED** [review]

**Pull request merged**